Solutions/Trend Micro Cloud App Security/Hunting Queries/TrendMicroCASVAThreats.yaml

```diff
@@ -0,0 +1,25 @@
+id: 5ce1415f-cdea-4740-a481-73c1394248c2
+name: Trend Micro CAS - Virtual Analizer threats
+description: |
+  'Query searches for Virtual Analizer threats.'

```

Comment:
Is there a typo in Analizer. Should it be analyzer.

Reply:
fixed